src/js/components/Tab/Tab.js

```diff
@@ -84,13 +95,26 @@ class Tab extends Component {
         onFocus={this.onMouseOver}
         onBlur={this.onMouseOut}
       >
-        <Box
-          pad={{ bottom: 'xsmall' }}
-          margin={{ vertical: 'xxsmall', horizontal: 'small' }}
-          border={{ side: 'bottom', size: 'small', color: borderColor }}
-        >
-          {normalizedTitle}
-        </Box>
+        {header ? (
+          <StyledTab
+            as={Box}
+            background={theme.tab.background}
+            theme={theme}
+          >
+            {header}
+          </StyledTab>
+        ) : (
+          <StyledTab
+            as={Box}
+            background={theme.tab.background}
+            theme={theme}
+            pad={{ bottom: 'xsmall' }}
+            margin={{ vertical: 'xxsmall', horizontal: 'small' }}
+            border={{ side: 'bottom', size: 'small', color: borderColor }}
```

Comment:
Would one nice if this came from the theme rather than here.

Reply:
checking...